docs/releases/ez_platform_v2.4.md

```diff
@@ -224,7 +224,7 @@ You are now able to load multiple Locations at once, using `LocationService->loa
                  # ...
     ```
 
-    4\. Make sure that parameter `page_builder.token_authenticator.enabled` has value `true`. If the parameter is not present, add it to `/app/config/config.yml`:
+    4\. Make sure that parameter `page_builder.token_authenticator.enabled` has value `true`. If the parameter is not present, add it to `/app/config/config.yaml`:
```

Comment:
`config/packages/ezplatform_page_builder.yaml`

Reply:
Release notes, so the changes should not have happened here at all.